### PR TITLE
Display a users owned rooms on their profile page.

### LINKED ISSUE #718: Show rooms owned by user on account page
When I go to my account page it would be nice to have a list of rooms I own listed.

### PR DESCRIPTION
**Pull request by hnrkndrssn**
Will show as:
image

The rooms are not links that you can click to open the room, I thought as this would open a new tab (i.e. not in the browser tab that JabbR is currently open in) you would end up with a number of tabs open.

But if others think that the rooms should be links opening the room, then I'll make the change.

 Fixes #718

### CODE REVIEW

**Comment by hnrkndrssn** [comment]
New layout

image

No note set
image

No note or flag set
image

File: JabbR/Content/themes/default/Chat.css [hunk 1]
```diff
@@ -438,9 +448,10 @@ canvas {
             color: rgba(0, 16, 63, 1);
         }
 
-#userlist-lobby-owners li div.room-row {
+#userlist-lobby-owners li div.room-row,
+#owned-rooms li div.room-row {
     display: inline-block;
-    width: 250px;
+    width: 240px;
```
Comment (cmaish) [inline JabbR/Content/themes/default/Chat.css]: This breaks locked and closed owned rooms in the lobby

**Comment by cmaish** [comment]
Width issue aside, looks good to me.  Lists the user's rooms.

I don't think that we need to make them links at this stage - maybe in the distant future.  Linkifying them would either require getting a target for the main jabbr window so we could do <a href="https://jabbr.net/#/rooms/meta" target="mainjabbrwindowtarget">Meta</a> (which I don't think is possible with javascript); or to connect to the chathub (which I think is what we're probably gonna have to eventually do in order to get realtime updates to flags, notes and rooms).

**Comment by hnrkndrssn** [comment]
Fixed the width issue on the Lobby

**Pull request merged**